projects/elife.yaml

```diff
@@ -745,6 +747,57 @@ generic-cdn:
             default-ttl: 86400 # seconds
             headers:
                 - Origin
+        fastly:
+            subdomains:
+                - "{instance}-cdn"
+            backends:
+                # first is default
+                default:
+                    hostname: elife-cdn.s3.amazonaws.com
+                articles:
+                    hostname: "{instance}-elife-published.s3.amazonaws.com"
+                    condition: req.url ~ "^/articles/"
+            default-ttl: 86400 # seconds
+            surrogate-keys:
+                article-id:
+                    url: "^/articles/([0-9]+)/(.+)$"
+                    value: "articles/\\1"
+    aws-alt:
+        # have to specify this because otherwise configuration such as `fresh` inherited from defaults won't see the ec2: false in `aws:` and test_validation.py will fail for all of them?
```

Comment:
urgh. at some point we should revisit the rough edges of project wrangling.

Reply:
Yes, in this specific case maybe the inherited `defaults` could be merged with the project's `aws`, but who knows what that could break.